rmgpy/molecule/filtration.py

```diff
@@ -104,12 +105,12 @@ def get_octet_deviation(mol, allow_expanded_octet=True):
     if `allow_expanded_octet` is ``True`` (by default), then the function also considers dectet for
     third row elements (currently sulfur is the only hypervalance third row element in RMG)
     """
-    if not isinstance(mol, Molecule):
+    if not isinstance(mol, (Molecule, Fragment)):
         raise TypeError("Octet deviation could only be determined for Molecule objects.")
```

Comment:
Minor: update the error message to include Fragment objects: `"Octet deviation could only be determined for Molecule or Fragment objects."`

Reply:
Done!